### ISSUE TITLE
Fedora 32 - Device not detected

### ISSUE THREAD

**Issue by didierga** [keredson/gnomecast#128]
Hi,

Running Fedora 32 Workstation I try to use gnomecast to cast to a ChromeCast Ultra but:

- gnomecast does not detect the ChromeCast
- gnomecast works fine if I specify IP of the ChromeCast
- google-chrome-stable-83.0.4103.116-1.x86_64 detects the ChromeCast with no problem 
- nmap find the ChromeCast

```
$ sudo nmap -sT 192.168.1.0/24
Starting Nmap 7.80 ( https://nmap.org ) at 2020-07-04 22:53 CEST
Nmap scan report for lan.home (192.168.1.1)
Host is up (0.0033s latency).
Not shown: 992 filtered ports
PORT    STATE  SERVICE
53/tcp  open   domain
80/tcp  open   http
113/tcp closed ident
135/tcp closed msrpc
139/tcp open   netbios-ssn
443/tcp open   https
445/tcp open   microsoft-ds
631/tcp open   ipp
MAC Address: 80:20:DA:XX:XX:XX (Sagemcom Broadband SAS)

Nmap scan report for Chromecast-Ultra.home (192.168.1.11)
Host is up (0.0068s latency).
Not shown: 995 closed ports
PORT      STATE SERVICE
8008/tcp  open  http
8009/tcp  open  ajp13
8443/tcp  open  https-alt
9000/tcp  open  cslistener
10001/tcp open  scp-config
MAC Address: 1C:F2:9A:7XX:XX:XX (Google)

Nmap scan report for laptop.home (192.168.1.25)
Host is up (0.00052s latency).
Not shown: 999 closed ports
PORT   STATE SERVICE
22/tcp open  ssh

Nmap done: 256 IP addresses (3 hosts up) scanned in 8.18 seconds
```

```
$ pip3 show gnomecast
Name: gnomecast
Version: 1.9.7
Summary: A native Linux GUI for Chromecasting local files.
Home-page: https://github.com/keredson/gnomecast
Author: Derek Anderson
Author-email: public@kered.org
License: UNKNOWN
Location: /usr/local/lib/python3.8/site-packages
Requires: pychromecast, bottle, pycaption, paste, html5lib
Required-by: 
```

Thanks for your help.

**Comment by didierga** [comment]
When I  start gnomecast in a terminal:

```
$ gnomecast
update_button_visible
None
update_button_visible
update_button_visible
serving on http://192.168.1.25:50639
check_ffmpeg
b'/usr/bin/ffmpeg\n'
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/gnomecast.py", line 559, in f
    friendly_name = cc.device.friendly_name
AttributeError: 'list' object has no attribute 'device'
```

**Comment by sandeep-sidhu** [comment]
I'm seeing the same problem.  I even downgraded to 1.9.5 and problem still exists.

**Comment by keredson** [comment]
What version of pychromecast are you using?

**Comment by sandeep-sidhu** [comment]
@keredson  pychromecast-7.1.1

other packages:   casttube-0.2.1 certifi-2020.6.20 chardet-3.0.4 idna-2.10 ifaddr-0.1.7 protobuf-3.12.2 pychromecast-7.1.1 requests-2.24.0 setuptools-49.1.0 six-1.15.0 urllib3-1.25.9 zeroconf-0.27.1

**Comment by didierga** [comment]
Same version for me:

```
$ pip3 show pychromecast
Name: PyChromecast
Version: 7.1.1
Summary: Python module to talk to Google Chromecast.
Home-page: https://github.com/balloob/pychromecast
Author: Paulus Schoutsen
Author-email: paulus@paulusschoutsen.nl
License: MIT
Location: /usr/local/lib64/python3.8/site-packages
Requires: protobuf, casttube, requests, zeroconf
Required-by: gnomecast

```

**Comment by keredson** [comment]
seems like a bug in the latest version of pychromecast.  i'm going to wait to see if they fix it.  if they don't want to then I'll add a shim work w/ both behaviors.
https://github.com/home-assistant-libs/pychromecast/issues/398

can downgrade pychromecast (to 7.1.0?) to work immediately if you wish.

**Comment by keredson** [comment]
pychromecast hasn't responded yet, so coded a workaround in https://github.com/keredson/gnomecast/commit/542528099407a77e0745069a7f6b5e6cdc0b9bd8.  released in 1.9.10.
